src/Theme/index.js

```diff
@@ -0,0 +1,33 @@
+const theme = {
+  fontSource: "https://fonts.googleapis.com/css?family=Poppins:400,600",
```

Comment:
instead of having source url, You can download the content from this source and keep in a file and then refer that file.